src/components/paperManager.js

```diff
@@ -1,5 +1,4 @@
 import { dia } from 'jointjs';
-import { highlightPadding } from '@/mixins/crownConfig';
```

Comment:
I did move crownConfig stuff to mixins/highlightConfig. If you want to do something like this instead:
```
import { highlightPadding } from '@/mixins/highlightConfig';
```

And place this in highlightConfig:
`export const highlightPadding = 3;`

It is only used in a few places, so either way is probably fine.

Reply:
I think I'll probably leave this 'as is' for now. I'm not convinced that these two padding amounts really represent the same thing and warrant being extracted to a single constant in this case.